src/components/Overlays/Overlays.fixture.tsx

```
@@ -0,0 +1,103 @@
+            <button>Hover me (children)</button>
+          </Tooltip>
+          some text ...
+        </p>
+      </div>
+
+      <div style={{ marginBottom: 20 }}>
+        <h4>Ref-based Trigger</h4>
+        <button ref={buttonRef}>Hover me (ref trigger)</button>
+        <Tooltip
+          content={'This is a tooltip with ref trigger.\n ' + FILLER_TEXT}
+          triggerRef={buttonRef}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)